sig-etcd/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG etcd Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Scope
+
+Owns the etcd project and how it is used by Kubernetes.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Development of [etcd] and other repositories under [etcd-io organization]
+- Maintenance of [etcd image] packaged with Kubernetes
+
+[etcd]: https://github.com/etcd-io/etcd
+[etcd-io organization]: https://github.com/etcd-io
+[etcd image]: https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd
+
+#### Cross-cutting and Externally Facing Processes
+
+- Specifying, testing and improving [The Implicit Kubernetes-ETCD Contract]
+- Release process of etcd and other binaries belonging to [etcd-io organization]
+
+[The Implicit Kubernetes-ETCD Contract]: https://docs.google.com/document/d/1NUZDiJeiIH5vo_FMaTWf0JtrQKCx0kpEaIIuPoj9P6A/edit?usp=sharing
+
+### Out of scope
+
+- Structure of data stored in etcd is owned by SIG API Machinery
+
+## Roles and Organization Management
+
+This SIG follows the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Tech Leads
+
+- Release of etcd and other binaries belonging to [etcd-io organization]
+
+### Deviations from [sig-governance]
+
+- SIG-Etcd's participation in the Kubernetes release cycle is limited by Etcd having a different schedule for its releases.  Particularly, Etcd may or may not participate in the Enhancements process depending on whether an upcoming Kubernetes release targets specific new Etcd features.
```

Comment:
Is participation in some of our processes (like KEPs) something that sig-etcd may look to do in the future? While it makes sense that etcd would have a different release cycle from the core Kubernetes itself (many subprojects do this too) there might be some benefit to looking at the processes like KEPs for things like PRRs

Reply:
Yes, we do intend to evaluate and adopt other kubernetes processes like PRRs in future. We have a doc to track the things that we will consider along the way: https://tinyurl.com/etcd-sig-checklist.

Reply:
@wenjiaswe Can we also make this info explicit here in the charter?

It's okay for etcd to not follow the Kubernetes release cycle right away. I think we have a few components which don't align to the Kubernetes release cycle. Participation in the KEP process doesn't necessarily imply adhering to the Kubernetes release cycle.

Reply:
Yes, following the release cycle is largely not a requirement for KEPs. Some of the review work like PRR may currently be influenced by the release cycle for practical reasons but etcd does not need to adhere to the kubernetes release lifecycle, KEPs or otherwise.

Most SIGs don't state in their charter which changes will involve KEPs and generally large or controversial changes trigger a request for a KEP by the approvers / OWNERS of projects, at their discretion.

So I don't think that needs determining here.

Reply:
Thank you for your input @BenTheElder @palnabarun . I agree that, even though `Etcd may or may not participate in the Enhancements process` is a valid statement and align with the rest of the SIGs, but `depending on whether an upcoming Kubernetes release targets specific new Etcd features` may cause some confusion, as SIG shouldn't have to adhere to the k8s release cycle to participate in the KEP process.

How about I just remove this statement to align with the charter of other sigs?